### PR TITLE
DT-1223: Add missing example value on displayedNameForXXX

### PR DESCRIPTION
**Pull request by HenrikHL**
[DT-1223](https://dcsa.atlassian.net/browse/DT-1223) - Add example values for the `displayedNameForXXX` properties

[DT-1223]: https://dcsa.atlassian.net/browse/DT-1223?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by alanb-dscsa — CHANGES_REQUESTED** [review]
Looks like there are now two examples for address line. Can you check it?

**Comment by HenrikHL** [comment]
@alanb-dscsa yes - there is a single line for the string and then there is the example (containing multiple lines) for the `displayedNameForXXX` examples

File: ebl/v3/issuance/EBL_ISS_v3.0.0.yaml [hunk 1]
```diff
@@ -609,6 +609,10 @@ components:
             description: |
               A line of the address to be displayed on the transport document.
             example: 'Strawinskylaan 4117'
+          example:
+            - Strawinskylaan 457
+            - 1077 XX Amsterdam
+            - Nederlandene
```
Comment (palatsangeetha) [inline ebl/v3/issuance/EBL_ISS_v3.0.0.yaml:615]: Is it Nederlanden?

Comment (HenrikHL) [reply ebl/v3/issuance/EBL_ISS_v3.0.0.yaml:615]: No idea - I copied it from Google...
image

**Comment by HenrikHL** [comment]
I will drop this PR - no examples will be added on `displayedNameForXXX` - only on property level

**Pull request closed without merging**